README.md

```diff
@@ -80,23 +80,23 @@ other transport related settings.
 To create a client to make API calls to Elasticsearch running on `http://localhost:9200`
 
 ```rust,no_run
-use elasticsearch::{Error, Elasticsearch};
+use elasticsearch::{Elasticsearch};
 
-fn run() {
-    let client = Elasticsearch::default();
+fn main() {
+    let _client = Elasticsearch::default();
 }
 ```
 Alternatively, you can create a client to make API calls against Elasticsearch running on a specific url
 
 ```rust,no_run
 use elasticsearch::{
-    Error, Elasticsearch, 
-    http::transport::{Transport, SingleNodeConnectionPool}
+    Error, Elasticsearch,
+    http::transport::{Transport}
 };
 
-fn run() -> Result<(), Error> {
+fn main() -> Result<(), Error> {
     let transport = Transport::single_node("https://example.com")?;
-    let client = Elasticsearch::new(transport);
+    let _client = Elasticsearch::new(transport);
```

Comment:
I realize client isn’t used, but would prefer to keep this as `client`, in line with all examples.